lib/script-view.js

```diff
@@ -18,6 +18,10 @@ export default class ScriptView extends MessagePanelView {
     this.scrollTimeout = null;
     this.ansiFilter = new AnsiFilter();
     this.headerView = headerView;
+    this.position = this.parsePositionConfig(atom.config.get('script.position'));
+    atom.config.observe('script.position', (newVal) => {
+      this.position = this.parsePositionConfig(newVal);
+    });
```

Comment:
atom.config.observe calls the config even when the config is not changed (once). So, if you want to set a default value, use a simple string. Later its value will be updated by `observe`

```suggestion
    this.position = 'bottom';
    atom.config.observe('script.position', (newVal) => {
      this.position = this.parsePositionConfig(newVal);
    });
```